### ISSUE TITLE
Update credits

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#81]
```
As it currently is, My (Joshua O'Sullivan) email is incorrect,
It should be "CoheedFTW@hotmail.co.uk"

I know, not the most important of changes, but I'd appreciate it. If you need 
to get in contact with me, email me and I can aid proving I own that email.

Thanks,
     ~ Josh / Dragonfly
```

-----
Original issue reported on code.google.com by `theclich...@gmail.com` on 6 Oct 2010 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
fixed in 3923:494221b5950e
```

-----
Original comment by `vittorio...@gmail.com` on 8 Oct 2010 at 3:48
* Changed state: **Fixed**
* Added labels: **Type-Other**, **Priority-Low**
* Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Verified (credits fixed with correct info)
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 8:07
* Changed state: **Verified**
